### ISSUE TITLE
IE 11 problem

### ISSUE THREAD

**Issue by rsmalc** [dangvanthanh/vue-ckeditor2#29]
In IE 11, CKEditor doesn't get content if it's exist. It shows only the empty editor. It shows permision error in console log.

**Comment by dangvanthanh** [comment]
@rsmalc Thank you very much. I will fix it soon.

**Comment by dangvanthanh** [comment]
@rsmalc 

I was fix Ckeditor doesn't get content in IE11

**Comment by usolved** [comment]
I'm using vue-ckeditor 1.18.0 where the fix is already commited but still getting "permission denied" error in IE11 console. I get this error even when IE 11 displays the content correctly.

So I could just ignore it but in some cases (like in bug #42) it also shows an empty editor when setting the data after ajax request for example.

**Comment by vicnoob** [comment]
I got this problem too, is there anyway to fix it?

**Comment by dangvanthanh** [comment]
Hi @vicnoob 

I will fix soon. Thank you very much for feedback.

**Comment by vicnoob** [comment]
@dangvanthanh It seems to be CKeditor's problem in general. I found in #42  that use :value can use in this situation, but it's 1 way binding, isn't it?

**Comment by dangvanthanh** [comment]
@vicnoob 

When use `v-model`, it will ignore `value` attributes on any form elements. This is two-way binding.

**Comment by vicnoob** [comment]
@dangvanthanh I know v-model is two-way binding, quoted from issue #42 

> Hey guys, i had the same issue.
Now work for me using :value instead of v-model

He's using :value which is one way binding to fix the problem. I am wondering if I use @input to make the instance 2 way binding again, will the problem happen again?

**Comment by vicnoob** [comment]
It seems that permission denied happened because Vue-ckeditor make the ckeditor into iframe? Is that right?

**Comment by vicnoob** [comment]
@dangvanthanh I tried to use pure CK editor, there is no error, it seems to be vue-ck2's error

**Comment by dangvanthanh** [comment]
@vicnoob 

Yes, It occur when using with `vue-ckeditor2` because I wrapper Ckeditor with Vue.js. I will try find error and fix it.

**Comment by vicnoob** [comment]
@dangvanthanh Thank you. It's urgent for me :(
